packages/cf-component-label/src/Label.js

```diff
@@ -1,16 +1,29 @@
 import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
 
-class Label extends React.Component {
-  render() {
-    return React.createElement(
-      this.props.tagName,
-      {
-        className: 'cf-label cf-label--' + this.props.type
-      },
-      this.props.children
-    );
-  }
-}
+const styles = props => {
+  const theme = props.theme;
+  return {
+    display: theme.display,
```

Comment:
I used the public styles, this was before we decided to have a singular style sheet. I'll put in our private styles.